### ISSUE TITLE
Website suggestion: remove landing page

### ISSUE THREAD

**Issue by jmeas** [davidkpiano/react-redux-form#1043]
The [landing page](https://davidkpiano.github.io/react-redux-form/) of the website doesn't provide much value, I think, and it also makes for a poor user experience to frequent users of this lib.

The animation might seem cool the first time, but if you're referencing the docs a lot, it can be annoying to have to wait for it each and every time before you can go to the docs.

I recommend that [this page](https://davidkpiano.github.io/react-redux-form/docs.html) be the first page that folks see. The benefits to this are:

1. More detailed links into the docs
2. Still has a GitHub link
3. The navigation is _immediately_ available

If you really want an animation somewhere, then perhaps the `/docs.html` page can include an animation, but not one that blocks the user from interacting with the navigation.

Thanks for reading!

**Comment by davidkpiano** [comment]
You're right, want to make a PR that redirects directly to the docs? Should be fairly simple, and I'll merge it in ASAP.

**Comment by jmeas** [comment]
Sure thing :v:

**Comment by jmeas** [comment]
I only had a moment to look at this, and I wasn't able to find the landing page in the source. I took a quick look in `/docs` and the root, and ran a search for the word "versatile," but nothing came up. In an effort to find it, I tried booting up the site locally, but `gitbook` isn't listed as a devDep, so it wouldn't start.

If anyone else wants to grab this, go for it. Otherwise, sometime in the next few days I'll:

- add `gitbook` to the devDeps
- find and remove the landing page

Thanks! And thanks for your quick reply @davidkpiano :v:
